### PR TITLE
Continue V3 UI integration work

### PR DESCRIPTION
**Pull request by offerijns**

### CODE REVIEW

File: src/actions/coordinator.ts [hunk 1]
```diff
@@ -39,34 +39,66 @@ export function CoordinatorActions<ActionsBase extends Constructor<TinlakeParams
       return { reserve, netAssetValue, seniorAsset, minTinRatio, maxTinRatio, maxReserve }
     }
 
+    getOrderState = async () => {
+      const coordinator = this.contract('COORDINATOR')
+      const orderState = await coordinator.order()
```
Comment (philipstanislaus) [inline src/actions/coordinator.ts:44]: Should this not live in tinlake.js?

File: src/actions/coordinator.ts [hunk 2]
```diff
@@ -95,21 +127,29 @@ export function CoordinatorActions<ActionsBase extends Constructor<TinlakeParams
     getCurrentEpochState = async () => {
       const coordinator = this.contract('COORDINATOR')
 
-      const submissionPeriod = await coordinator.submissionPeriod()
-      if (!submissionPeriod) return 'open'
-
-      const minChallengePeriodEnd = await coordinator.minChallengePeriodEnd()
+      const minChallengePeriodEnd = (await coordinator.minChallengePeriodEnd()).toBN().toNumber()
+      const latestBlockTimestamp = (await this.provider.getBlock(await this.provider.getBlockNumber())).timestamp
```
Comment (philipstanislaus) [inline src/actions/coordinator.ts:131]: Nice! 👍

File: src/actions/lender.ts [hunk 3]
```diff
@@ -4,16 +4,17 @@ import BN from 'bn.js'
 export function LenderActions<ActionBase extends Constructor<TinlakeParams>>(Base: ActionBase) {
   return class extends Base implements ILenderActions {
     // senior tranche functions
-    submitSeniorSupplyOrder = async (user: string, supplyAmount: string) => {
+    submitSeniorSupplyOrder = async (supplyAmount: string) => {
       return this.pending(this.contract('SENIOR_OPERATOR').supplyOrder(supplyAmount, this.overrides))
```
Comment (philipstanislaus) [inline src/actions/lender.ts:8]: Again, this functions (and others from this PR) should go into Tinlake.js, right?

Comment (offerijns) [reply src/actions/lender.ts:8]: What do you mean? This is Tinlake.js?

Comment (philipstanislaus) [reply src/actions/lender.ts:8]: Haha, it’s getting late - saw UI in the title and just looked at the code 🙈

**Pull request merged**